### DISCUSSION TITLE
Remove some preset tiles

### DISCUSSION THREAD

**Discussion by CZonin** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#524] [Q&A] [answered]
Is it possible to remove some of the preset tiles other than by configuring each one manually?

**Comment by PiotrMachowski** [comment, marked as answer]
You can:
 - click "generate static config" in card editor and remove unwanted tiles
 - click "generate static config", add `append_tiles: true`, leave just config of an unwanted tile and add to it a condition that will always be evaluated to false

In both cases remove other generated sections

**Reply by saschabrockel** [reply]
I have been searching for 15 minutes now on how to adjust the tiles. Can you put this in the documentation/readme? It would help many people.

**Comment by CZonin** [comment]
Didn't realize what that option did in the editor, thanks!
